### PR TITLE
Support for BSM registry management web services

### PR DESCRIPTION
**Pull request by teodoradobreva**

### CODE REVIEW

File: src/04 - Web services.md [hunk 1]
```diff
@@ -103,58 +116,230 @@ Content-Type: application/json
...
+###### Details
+
+- **groupId**
+  - group ID of the group definition in which to create or update the field definition
+- **fieldId**
+  - field ID of the field definition to be created or updated
+- **fieldName**
+  - field name
+- **fieldDescription**
+  - group description
```
Comment (douzounov) [inline src/04 - Web services.md]: I think this should read `field description`.

Comment (teodoradobreva) [reply src/04 - Web services.md]: Fixed.

File: src/04 - Web services.md [hunk 2]
```diff
@@ -103,58 +116,230 @@ Content-Type: application/json
...
+      - system unique field ID (a string comprised of lower-case letters, digits, and dashes that must begin with a letter and end with a letter or a digit)
+    - **fieldName**
+      - field name
+    - **fieldDescription**
+      - field description
+    - **fieldType**
+      - field type (possible values are string, long, and double)
+    - **fieldValues** (optional)
+      - array of field values of the particular field type
+      - will be returned only for controlled fields (an array of zero or more values); will not be returned for simple fields
```
Comment (ivaylomitrev) [inline src/04 - Web services.md]: I would assume the terms `controlled fields` and `simple fields` would mean little to integrating parties.

I'd suggest re-wording this to:
```
- if present, an array of zero or more values
- will be returned only for fields with pre-defined values (i.e. values assigned to the field upon its creation)
```

The same occurs in at least one more location in this document too.

Comment (teodoradobreva) [reply src/04 - Web services.md]: Fixed.

**Comment by teodoradobreva** [comment]
Squashed the to fixes only.

**Pull request merged**